src/Log4netShipper/Log4netShipper.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks Condition=" '$(TargetFrameworks)' == '' ">netcoreapp3.1,netstandard2.0;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```

Comment:
Think it is sad that you have removed netstandard2.0 and no longer supports NetFramework. Maybe split this pull-request into 2 pull-requests. One that updates the build-pipeline. And one that adds `EnableJsonMessage`.